src/cromshell/utilities/cromshellconfig.py

```diff
@@ -245,6 +247,40 @@ def resolve_requests_connect_timeout(timeout_cli: int):
         LOGGER.info("Request Timeout value: %d sec", requests_connect_timeout)
 
 
+def resolve_referer_header_url(url: str):
+
+    global referer_header_url
+
+    if url:
+        LOGGER.info(f"Will send referer header {url} from command line options.")
```

Comment:
Nit: These feel more debug level than info to me. Especially the config-file-fetched one. And *especially* especially the "no referer header URL set" one.

Reply:
I agree in general but I am following Cromshell's apparent house style of narrating what it's doing and reading back parameters. Because it's a run & done tool the log stays a manageable length.

```
> cromshell-alpha submit /Users/anichols/Library/CloudStorage/OneDrive-Personal/Broad-Institute/Bugs/BW-1192/input-task.wdl /Users/anichols/Library/CloudStorage/OneDrive-Personal/Broad-Institute/Bugs/BW-1192/input-task.json
2022-05-02 19:50:55,085 cromshell.__main__                         INFO     Invoked by: /Users/anichols/.pyenv/versions/3.10.3/bin/cromshell-alpha submit /Users/anichols/Library/CloudStorage/OneDrive-Personal/Broad-Institute/Bugs/BW-1192/input-task.wdl /Users/anichols/Library/CloudStorage/OneDrive-Personal/Broad-Institute/Bugs/BW-1192/input-task.json
2022-05-02 19:50:55,085 cromshell.__main__                         INFO     Log level set to: INFO
2022-05-02 19:50:55,085 cromshell.utilities.cromshellconfig        INFO     Workflow id and cromwell server not specified. Using default cromwell server 
2022-05-02 19:50:55,085 cromshell.utilities.cromshellconfig        INFO     Server: https://leonardo.dsde-dev.broadinstitute.org/proxy/google/v1/apps/terra-dev-ae321a61/terra-app-31ba3203-d27e-4886-b128-39e720649ed1/cromwell-service/
2022-05-02 19:50:55,085 cromshell.utilities.cromshellconfig        INFO     Setting requests timeout from value in config file.
2022-05-02 19:50:55,085 cromshell.utilities.cromshellconfig        INFO     Request Timeout value: 5 sec
2022-05-02 19:50:55,085 cromshell.utilities.cromshellconfig        INFO     Will send auth header with token for oednichols@gmail.com from config.
2022-05-02 19:50:55,085 cromshell.utilities.cromshellconfig        INFO     Will send referer header https://leonardo.dsde-dev.broadinstitute.org from config.
2022-05-02 19:50:55,086 cromshell.submit.command                   INFO     submit
2022-05-02 19:50:57,026 cromshell.submit.command                   INFO     Submitting job to server: https://leonardo.dsde-dev.broadinstitute.org/proxy/google/v1/apps/terra-dev-ae321a61/terra-app-31ba3203-d27e-4886-b128-39e720649ed1/cromwell-service/

                 __
      .,-;-;-,. /'_\
    _/_/_/_|_\_\) /
  '-<_><_><_><_>=/\
    `/_/====/_/-'\_\
     ""     ""    ""
            
{
    "id": "94782959-8b2a-48f7-949d-25a4972820db",
    "status": "Submitted"
}
```

There is also a `--quiet` option that I verified correctly applies to my `INFO` additions.

```
> cromshell-alpha --quiet status ba888f6e-3d7b-47b9-9480-f1243e1b5c9b

                 __
      .,-;-;-,. /'_\
    _/_/_/_|_\_\) /
  '-<_><_><_><_>=/\
    `/_/====/_/-'\_\
     ""     ""    ""
            
{"status":"Succeeded",
"id":"ba888f6e-3d7b-47b9-9480-f1243e1b5c9b"}
```